vendor.conf

```diff
@@ -6,7 +6,7 @@ github.com/davecgh/go-spew v1.1.0
 github.com/pmezard/go-difflib v1.0.0
 golang.org/x/sys 9aade4d3a3b7e6d876cd3823ad20ec45fc035402 # update to 7ddbeae9ae08c6a06a59597f0c9edbc5ff2444ce wiht pkg/signal change
```

Comment:
Can we sync this to containerd now?

Reply:
I don't remember the details for this. I assume `pkg/signal` didn't build with updated version. If it builds then we can update, if it doesn't we may need to do a PR to moby first.

Reply:
updated, compilation seems ok